src/main/java/org/mastodon/mamut/feature/branch/BranchNSubBranchSpotsFeature.java

```diff
@@ -0,0 +1,110 @@
+package org.mastodon.mamut.feature.branch;
+
+import static org.mastodon.feature.FeatureProjectionKey.key;
+
+import java.util.Collections;
+import java.util.Set;
+
+import org.mastodon.feature.Dimension;
+import org.mastodon.feature.Feature;
+import org.mastodon.feature.FeatureProjection;
+import org.mastodon.feature.FeatureProjectionKey;
+import org.mastodon.feature.FeatureProjectionSpec;
+import org.mastodon.feature.FeatureProjections;
+import org.mastodon.feature.FeatureSpec;
+import org.mastodon.feature.IntFeatureProjection;
+import org.mastodon.feature.Multiplicity;
+import org.mastodon.mamut.model.branch.BranchSpot;
+import org.mastodon.properties.IntPropertyMap;
+import org.scijava.plugin.Plugin;
+
+/**
+ * Represents the total number of successors of a branch spot in the whole track sub-tree of this branch spot.
+ * <p>
+ * In the following example this number would equal to following branchSpots as
+ * follows:
+ *
+ * <pre>
+ *                         branchSpot0
+ *  	       ┌──────────────┴─────────────────┐
+ *  	       │                                │
+ *  	   branchspot1                      branchSpot2
+ *  	┌──────┴───────┐
+ *  	│              │
+ *  branchspot3 branchSpot4
+ * </pre>
+ *
+ * <ul>
+ * <li>{@code branchSpot0 = 4}</li>
+ * <li>{@code branchSpot1 = 2}</li>
+ * <li>{@code branchSpot2 = 0}</li>
+ * <li>{@code branchSpot3 = 0}</li>
+ * <li>{@code branchSpot4 = 0}</li>
+ * </ul>
+ */
+public class BranchNSubBranchSpotsFeature
+		implements Feature< BranchSpot >
+{
+	public static final String KEY = "Branch N sub branch spots";
+
+	private static final String HELP_STRING =
+			"Computes the total number of successors of a branch spot in the whole track sub-tree of this branch spot.";
```

Comment:
Please simplify to "Computes the number of successors of a branch spot."

Reply:
Or "number of descendants"

Reply:
To make it consistent with BranchNLeavesFeature, I have set it to: "Counts the successors in the sub-tree of this branch spot."